app/javascript/packs/search.js

```diff
@@ -39,7 +39,9 @@ function checkboxesFromUrl(elementName) {
 }
 
 function publicationDateValidation(){
-  if ($("#starting_year").val() > $("#ending_year").val()) {
+  startYear = parseInt($("#starting_year").val())
+  endYear = parseInt($("#ending_year").val())
+  if (startYear > endYear) {
     $("#starting_year")[0].setCustomValidity("Starting year should be lower than the ending year.");
```

Comment:
I think "earlier" or "before" might be more clear than "lower" since we are talking about dates.